validation-test/Python/copyright-conformance.swift

```diff
@@ -0,0 +1,3 @@
+// Checks to make sure all copyright headers conform with their
+// git creation/last modification date for silly legal reasons.
```

Comment:
Please remove the text "for silly legal reasons".  Only lawyers are allowed to opine on whether legal issues are silly or not.  😄

Reply:
quite sage

committed 👍